.github/dependency_tests.yaml

```
@@ -0,0 +1,12 @@
+broker[docker]: "tests/foreman/ -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'"
+deepdiff: "tests/foreman/endtoend/test_api_endtoend.py -k 'test_positive_get_links'"
+dynaconf[vault]: "tests/foreman/api/test_ldapauthsource.py -k 'test_positive_endtoend'"
+manifester: "tests/foreman/api/test_subscription.py -k 'test_positive_create_after_refresh'"
+navmazing: "tests/foreman/ui/test_repository.py -k 'test_positive_create_as_non_admin_user'"
+pyotp: "tests/foreman/ui/test_ldap_authentication.py -k 'test_positive_login_user_password_otp'"
+pytest-xdist: "tests/foreman/ -n 3 -m 'build_sanity'"
```

I think it'd be better to run this with 2 workers, wdyt?
```suggestion
pytest-xdist: "tests/foreman/ -n 2 -m 'build_sanity'"
```

---

That's fine I guess it has to be more than 1.

---

yeah, but less the better as it will checkout this number of satellites